readme.md

```diff
@@ -26,8 +25,7 @@ gulp.task('default', () => {
 			svgoPlugins: [
 				{removeViewBox: false},
 				{cleanupIDs: false}
-			],
-			use: [pngquant()]
```

Comment:
Removed this example since it'll confuse users and they'll complain when only their PNGs gets optimized.